test/clj/harja/palvelin/palvelut/budjettisuunnittelu_test.clj

```diff
@@ -194,6 +197,37 @@
                                                  (range 1 6)))
                                    (into #{} (keys (group-by (juxt :hoitokausi :maksukausi) tiedot)))))))))))
 
+(deftest indeksikertoimien-haku
+  (let [rovaniemi-urakka-id (hae-rovaniemen-maanteiden-hoitourakan-id)
+        ivalo-urakka-id (hae-ivalon-maanteiden-hoitourakan-id)
+        pellon-urakka-id (hae-pellon-maanteiden-hoitourakan-id)
+        ;; Indeksi lyödään automaagisesti urakalle, josta syystä Pellolla saattaa olla vanha indeksi käytössä
+        _ (u (str "UPDATE urakka SET indeksi = 'MAKU 2015' WHERE id = " pellon-urakka-id ";"))
+        kuluvan-hoitokauden-aloitusvuosi (-> (pvm/nyt) pvm/paivamaaran-hoitokausi first pvm/vuosi)
+
+        db (:db jarjestelma)
+
+        rovaniemen-indeksit (bs/hae-urakan-indeksikertoimet db +kayttaja-jvh+ {:urakka-id rovaniemi-urakka-id})
+        ivalon-indeksit (bs/hae-urakan-indeksikertoimet db +kayttaja-jvh+ {:urakka-id ivalo-urakka-id})
+        pellon-indeksit (bs/hae-urakan-indeksikertoimet db +kayttaja-jvh+ {:urakka-id pellon-urakka-id})]
+    (clojure.pprint/pprint ivalon-indeksit)
+    (clojure.pprint/pprint pellon-indeksit)
+    (is (= rovaniemen-indeksit ivalon-indeksit) "Indeksit pitäisi olla sama samaan aikaan alkaneille urakoillle")
+    (is (= rovaniemen-indeksit [{:vuosi kuluvan-hoitokauden-aloitusvuosi :indeksikerroin 1.000255}
```

Comment:
Urakan aloitusvuosi muuttuu dynaamisesti, mutta testin tulostarkistus on kovakoodattu.
Pitäis laskea kertoimet dynaamisesti, jos haluaa jättää testin tähän.